### PR TITLE
Doc/add generic docs

### PR DESCRIPTION
**Pull request by szuecs**

### CODE REVIEW

**Comment by szuecs** [comment]
trigger zappr

File: README.md [hunk 1]
```diff
@@ -76,10 +76,31 @@ however they depend on your OS. E.g. if you run the tests on **Linux** it will
 test the implementation in `keyring_linux.go` and similar if running the tests
 in **OS X** it will test the implementation in `keyring_darwin.go`.
 
+## Contributing/TODO
+
+We welcome contributions from the community. Please use [CONTRIBUTING.md](CONTRIBUTING.md) as guideline. To help you get started, here are some items that we'd love help with:
+
+- [Windows support](https://github.com/zalando/go-keyring/issues/3)
+- [Travis OS X](https://github.com/zalando/go-keyring/issues/2)
```
Comment (mikkeloscar) [inline README.md]: I already have a branch with this, will make a PR now :)

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**